R/bootstrap_parameters.R

```diff
@@ -46,7 +59,15 @@ bootstrap_parameters <- function(model, iterations = 1000, centrality = "median"
     p_value <- FALSE
   }
 
-  parameters <- bayestestR::describe_posterior(data, centrality = centrality, ci = ci, ci_method = ci_method, test = test, ...)
+  parameters <-
```

Comment:
Not sure if we can add a style-rule here, but I find it odd to have a line-break after the assignment arrow. I would rather format into something like:

```r
parameters <- bayestestR::describe_posterior(
  data,
  centrality = centrality,
  ci = ci,
  ci_method = ci_method,
  test = test,
  ...
)
```